### ISSUE TITLE
Remove beta-flag?

### ISSUE THREAD

**Issue by enkelmedia** [ligershark/WebOptimizer.Sass#47]
Would you consider moving this package from beta, not sure if the flag is still needed?

**Comment by madskristensen** [comment]
Yes, no need for the beta flag anymore. Would you consider sending a PR?

**Comment by enkelmedia** [comment]
Sweet! Sure thing! I'll have a look tomorrow =D
